README.rst

```diff
@@ -1,6 +1,46 @@
 Demos
 =====
-A collection of coding examples using the D-Wave Ocean SDK.
+A collection of demos using the D-Wave Ocean SDK.
+
+**NOTE** Each demo is stored in its own subdirectory and may have its own
+set of package requirements (for example, `antenna-selection/requirements.txt
+<antenna-selection/requirements.txt>`_).
+
+Setup Instructions
+------------------
+As each demo may have a different set of package requirements, you may want to
+set up a virtual environment to contain said packages and run your demo within
+that environment.
+
+Virtual environment setup and package installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+::
+
+  git clone git@github.com:dwavesystems/demos.git
+  cd demos/<desired-demo>
+  python3 -m venv <desired-demo>_env
+  source <desired-demo>_env/bin/activate
+  pip install -r requirements.txt
+
+Once the commands above have created and activated your virtual environment,
+and have installed the package requirements, you can run your demo.
+
+Virtual environment deactivation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Once you're ready to leave your virtual environment, simply type:
+::
+  deactivate
+
+When you want to run your demo again, remember to activate your virtual
+environment first:
+::
+  source <desired-demo>_env/bin/activate
```

Comment:
Actually, if you want to be shell/OS-inclusive, `source` is a Bashism. POSIX command for sourcing is `.`.

Reply:
lol! I know you're saying that the POSIX command for sourcing is `.` but it just looks like you're just trailing off your sentence.

I had initially written `source` because what was happening would seem more expressive to a new user. I didn't know one was a bashism and the other POSIX; I had always assumed that `.` was just a shorthand for `source`. Thanks for clarifying that :)